src/main/drivers/serial_escserial.c

```diff
@@ -181,11 +181,7 @@ static void escSerialGPIOConfig(const timerHardware_t *timhw, ioConfig_t cfg)
     }
 
     IOInit(IOGetByTag(tag), OWNER_MOTOR, 0);
-#if defined(STM32F7) || defined(STM32H7)
     IOConfigGPIOAF(IOGetByTag(tag), cfg, timhw->alternateFunction);
```

Comment:
@blckmn : Can you check this again? Old version failed to set MUX for `escSerialInputPortConfig` (escSerialGPIOConfig(timerHardwarePtr, IOCFG_AF_PP_UP); )
Maybe invalid MUX value was masking other problem. 
Or old code kept old mux settings (gpio_init does not change it)


> The call to IOConfigGPIO(IOGetByTag(tag), cfg); simply results in a call to IOConfigGPIOAF(IOGetByTag(tag), cfg, timhw->alternateFunction); but with AF provided as 0.

This is not true for AT32